sklearn/cluster/dbscan_.py

```diff
@@ -177,8 +177,17 @@ class DBSCAN(BaseEstimator, ClusterMixin):
         If metric is "precomputed", X is assumed to be a distance matrix and
         must be square.
     random_state : numpy.RandomState, optional
-        The generator used to shuffle the samples. Defaults to numpy.random.
-
+        The generator used to initialize the centers. Defaults to numpy.random.
```

Comment:
I think I did this to make it consistent with other clusterers may be... Was the old one better?